cMake/FindOCC.cmake

```
@@ -7,164 +7,134 @@
     set(OCC_LIBRARIES ${OpenCASCADE_LIBRARIES})
     set(OCC_OCAF_LIBRARIES TKCAF TKXCAF)
-  else(OpenCASCADE_FOUND)
-    if(WIN32)
-      if(CYGWIN OR MINGW)
-      FIND_PATH(OCC_INCLUDE_DIR Standard_Version.hxx
-          /usr/include/opencascade
-          /usr/local/include/opencascade
-          /opt/opencascade/include
-          /opt/opencascade/inc
-        )
-        FIND_LIBRARY(OCC_LIBRARY TKernel
-          /usr/lib
-          /usr/local/lib
-          /opt/opencascade/lib
-        )
-      else(CYGWIN OR MINGW)
-      FIND_PATH(OCC_INCLUDE_DIR Standard_Version.hxx
-          "[HKEY_LOCAL_MACHINE\\SOFTWARE\\SIM\\OCC\\2;Installation Path]/include"
-        )
-        FIND_LIBRARY(OCC_LIBRARY TKernel
-          "[HKEY_LOCAL_MACHINE\\SOFTWARE\\SIM\\OCC\\2;Installation Path]/lib"
+else ()
+    if (WIN32)
+        if (CYGWIN OR MINGW)
+            find_path(OCC_INCLUDE_DIR Standard_Version.hxx
+                    /usr/include/opencascade
+                    /usr/local/include/opencascade
+                    /opt/opencascade/include
+                    /opt/opencascade/inc
+            )
+            find_path(OCC_LIBRARY TKernel
+                    /usr/lib
+                    /usr/local/lib
+                    /opt/opencascade/lib
+            )
+        else ()
+            find_path(OCC_INCLUDE_DIR Standard_Version.hxx
+                    "[HKEY_LOCAL_MACHINE\\SOFTWARE\\SIM\\OCC\\2;Installation Path]/include"
+            )
+            find_library(OCC_LIBRARY TKernel
+                    "[HKEY_LOCAL_MACHINE\\SOFTWARE\\SIM\\OCC\\2;Installation Path]/lib"
+            )
+        endif ()
+    else ()
+        find_path(OCC_INCLUDE_DIR Standard_Version.hxx
+                /usr/include/occt
+                /usr/include/opencascade
+                /usr/local/include/opencascade
+                /opt/opencascade/include
+                /opt/opencascade/inc
         )
-      endif(CYGWIN OR MINGW)
-    else(WIN32)
-      FIND_PATH(OCC_INCLUDE_DIR Standard_Version.hxx
-        /usr/include/occt
-        /usr/include/opencascade
-        /usr/local/include/opencascade
-        /opt/opencascade/include
-        /opt/opencascade/inc
-      )
-      FIND_LIBRARY(OCC_LIBRARY TKernel
-        /usr/lib
-        /usr/local/lib
-        /opt/opencascade/lib
-      )
-    endif(WIN32)
-    if(OCC_LIBRARY)
-      GET_FILENAME_COMPONENT(OCC_LIBRARY_DIR ${OCC_LIBRARY} PATH)
-      IF(NOT OCC_INCLUDE_DIR)
-        FIND_PATH(OCC_INCLUDE_DIR Standard_Version.hxx
-          ${OCC_LIBRARY_DIR}/../inc
+        find_library(OCC_LIBRARY TKernel
+                /usr/lib
+                /usr/local/lib
+                /opt/opencascade/lib
         )
-      ENDIF()
-    endif(OCC_LIBRARY)
-  endif(OpenCASCADE_FOUND)
-endif(OCE_FOUND)
+    endif ()
+    if (OCC_LIBRARY)
+        get_filename_component(OCC_LIBRARY_DIR ${OCC_LIBRARY} PATH)
+        if (NOT OCC_INCLUDE_DIR)
+            FIND_PATH(OCC_INCLUDE_DIR Standard_Version.hxx
```

As long as you're normalizing the capitalization (and thank you for that!), may as well grab this one too:
```suggestion
            find_path(OCC_INCLUDE_DIR Standard_Version.hxx
```